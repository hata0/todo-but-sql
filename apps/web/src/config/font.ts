export const font = {
  display:
    "font-noto-sans-jp font-normal text-[32px]/[40px] tracking-[0px] md:text-[45px]/[52px] md:tracking-[0px] lg:text-[57px]/[64px] lg:-tracking-[0.25px]",
  heading:
    "font-noto-sans-jp font-normal text-[24px]/[32px] tracking-[0px] md:text-[28px]/[36px] md:tracking-[0px] lg:text-[32px]/[40px] lg:tracking-[0px]",
  title:
    "font-noto-sans-jp font-medium text-[14px]/[20px] tracking-[0.1px] md:text-[16px]/[24px] md:tracking-[0.15px] lg:font-normal lg:text-[22px]/[28px] lg:tracking-[0px]",
  body: "font-noto-sans-jp font-normal text-[12px]/[16px] tracking-[0.4px] md:text-[14px]/[20px] md:tracking-[0.25px] lg:text-[16px]/[24px] lg:tracking-[0.5px]",
  label:
    "font-noto-sans-jp font-medium text-[11px]/[16px] tracking-[0.5px] md:text-[12px]/[16px] md:tracking-[0.5px] lg:text-[14px]/[20px] lg:tracking-[0.1px]",
};
