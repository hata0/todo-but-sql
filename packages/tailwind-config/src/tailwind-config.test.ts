import { describe, expect, it } from "vitest";
import config from "./tailwind-config";

describe("tailwind.config.ts", () => {
  it("config", () => {
    expect(config).toMatchSnapshot();
  });
});
