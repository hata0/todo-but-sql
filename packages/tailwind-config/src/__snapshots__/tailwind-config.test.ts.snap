// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`tailwind.config.ts > config 1`] = `
{
  "plugins": [],
  "theme": {
    "borderRadius": {
      "16": "16px",
      "24": "24px",
      "4": "4px",
      "8": "8px",
      "full": "9999px",
      "none": "0px",
    },
    "brightness": {
      "hover-focus": "0.95",
      "press": "0.9",
    },
    "colors": {
      "accent": {
        "DEFAULT": "hsl(var(--accent))",
        "foreground": "hsl(var(--accent-foreground))",
      },
      "background": "hsl(var(--background))",
      "border": "hsl(var(--border))",
      "card": {
        "DEFAULT": "hsl(var(--card))",
        "foreground": "hsl(var(--card-foreground))",
      },
      "chart": {
        "1": "hsl(var(--chart-1))",
        "2": "hsl(var(--chart-2))",
        "3": "hsl(var(--chart-3))",
        "4": "hsl(var(--chart-4))",
        "5": "hsl(var(--chart-5))",
      },
      "destructive": {
        "DEFAULT": "hsl(var(--destructive))",
        "foreground": "hsl(var(--destructive-foreground))",
      },
      "foreground": "hsl(var(--foreground))",
      "input": "hsl(var(--input))",
      "muted": {
        "DEFAULT": "hsl(var(--muted))",
        "foreground": "hsl(var(--muted-foreground))",
      },
      "popover": {
        "DEFAULT": "hsl(var(--popover))",
        "foreground": "hsl(var(--popover-foreground))",
      },
      "primary": {
        "DEFAULT": "hsl(var(--primary))",
        "foreground": "hsl(var(--primary-foreground))",
      },
      "ring": "hsl(var(--ring))",
      "secondary": {
        "DEFAULT": "hsl(var(--secondary))",
        "foreground": "hsl(var(--secondary-foreground))",
      },
    },
    "fontFamily": {
      "noto-sans-jp": "var(--font-noto-sans-jp)",
    },
    "fontSize": {
      "12": [
        "12px",
        "20px",
      ],
      "14": [
        "14px",
        "22px",
      ],
      "16": [
        "16px",
        "24px",
      ],
      "20": [
        "20px",
        "28px",
      ],
      "32": [
        "32px",
        "40px",
      ],
    },
    "opacity": {
      "disabled": "0.38",
    },
    "spacing": {
      "0": "0px",
      "104": "104px",
      "16": "16px",
      "168": "168px",
      "24": "24px",
      "272": "272px",
      "4": "4px",
      "40": "40px",
      "440": "440px",
      "64": "64px",
      "8": "8px",
    },
    "zIndex": {
      "0": "0",
      "1": "1",
      "2": "2",
      "3": "3",
    },
  },
}
`;
